src/core/sys/posix/sys/time.d

```diff
@@ -28,7 +28,55 @@ else version (WatchOS)
     version = Darwin;
 
 version (Posix):
-extern (C) nothrow @nogc:
+nothrow @nogc:
+
+
+/* macros provided to operate on timeval structures
+ * they are extern(D) because they are not really C symbols, just macros
+ */
```

Comment:
It's a nit, and not critical. But it does allow you to avoid attribute verbosity.